README.md

```diff
@@ -85,43 +91,48 @@ Please refer to [TensorFlow installation page](https://www.tensorflow.org/instal
 When TensorFlow 2.0 and/or PyTorch has been installed, you can install from source by cloning the repository and running:
 
 ```bash
+git clone https://github.com/huggingface/transformers
+cd transformers
 pip install [--editable] .
 ```
 
+When you update the repository, you should upgrade the transformers installation and its dependencies as follows:
+
+```bash
+git pull
+pip install --upgrade [--editable] .
```

Comment:
Shouldn't we _always_ preconize the `--editable` flag? (cc @LysandreJik)

Reply:
Users who want to run the examples don't need it _if they re-run `pip install` every time they fetch changes from the repository_.

Contributors definitely need it. They also need to re-run `pip install` when dependencies change i.e. rarely.

I'm clarifying this.

Reply:
is there any drawback to always doing it? (even for casual users)

Reply:
A non-editable install is a "standard" install, similar to what you would get with `pip install transformers` from PyPI.

If everything goes well (#334 suggests that sometimes not everything goes well), an editable install will achieve the same results. The differences I can think about are minor:

- if a user accidentally makes changes to the source code, this will affect an editable install, while a non-editable install would still work ;
- see also the pip bug that I had to work around; you don't want users to run into such bugs, which are more likely with editable installs.

I didn't mention editable installs in the README because I was trying to make the best recommendation for the target audience, which has two types of users:

1. Those who don't know or care about editable installs. The documentation is primarily targeted at them. Let's keep things simple for them! Simply showing the option raises the questions of "what does this option do?" and "should I care about this?"
2. Those who understand editable installs. They'll know to use the editable option anyway if they need it, so you hardly need to document it.

So, to sum up — keep it simple :-)